tests/suite-integration/PluginFlyvemdmAgent.php

```diff
@@ -612,7 +612,7 @@ public function testPingRequest() {
       $logEntryFound = 0;
       $rows = $log->find("`direction` = 'O' AND `id` > '$lastLogId'");
       foreach ($rows as $row) {
-         if ($row['topic'] == $agent->getTopic() . '/Command/Ping') {
+         if ($row['topic'] == 'Command/Ping') {
```

Comment:
without this PR the tests select the log entry for the tested agent. Now, the topic does not contains this information. I think the unit  test is degraded by your change. We should add some assertions to ensure we will not wrongly pick a message for an other item.